internal/config/config.go

```diff
@@ -43,7 +43,9 @@ type Config struct {
 	KafkaDispatcherTopic    string
 	KafkaGroupID            string
 	KafkaInventoryTopic     string
+	KafkaPassword			string
```

Comment:
I noticed here that the type `string` doesn't line up with the line above it. When I inspected the files directly, it appears this line is using tabs rather than spaces to align the type declaration. Go formatting prefers spaces to tabs. `go fmt` should take care of whitespace issues like this, as well as additional, basic formatting. Please run `go fmt` on the various files you've changed before committing them. An IDE like Visual Studio Code with the Go extension will automatically format files when they are saved, saving you from having to manually run `go fmt` on your source files.